studio/src/pages/[organizationSlug]/[namespace]/graph/[slug]/overrides.tsx

```diff
@@ -0,0 +1,217 @@
+} from "@/components/ui/table";
+import { cn } from "@/lib/utils";
+import { useApplyParams } from "@/components/analytics/use-apply-params";
+import { ConfigureOverride } from "@/components/checks/override";
+import { formatDistanceToNow } from "date-fns";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { createFilterState } from "@/components/analytics/constructAnalyticsTableQueryState";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from "@/components/ui/tooltip";
+import { BiAnalyse } from "react-icons/bi";
+import { IoBarcodeSharp } from "react-icons/io5";
+
+const OverridesPage: NextPageWithLayout = () => {
+  const graphContext = useContext(GraphContext);
+  const router = useRouter();
+
+  const organizationSlug = router.query.organizationSlug as string;
+  const namespace = router.query.namespace as string;
+  const slug = router.query.slug as string;
+
+  const constructLink = (
+    name: string,
+    hash: string,
+    mode: "metrics" | "traces",
+  ) => {
+    const filterState = createFilterState({
+      operationName: name,
+      operationHash: hash,
+    });
+
+    if (mode === "metrics") {
+      return `/${organizationSlug}/${namespace}/graph/${slug}/analytics?filterState=${filterState}`;
+    }
+
+    return `/${organizationSlug}/${namespace}/graph/${slug}/analytics/traces?filterState=${filterState}`;
+  };
+
+  const { data, isLoading, error, refetch } = useQuery({
+    ...getAllOverrides.useQuery({
+      graphName: graphContext?.graph?.name,
+      namespace: graphContext?.graph?.namespace,
+    }),
+    enabled: !!graphContext?.graph?.name,
+  });
+
+  const applyParams = useApplyParams();
+
+  if (isLoading) return <Loader fullscreen />;
+
+  if (error || data?.response?.code !== EnumStatusCode.OK)
+    return (
+      <EmptyState
+        icon={<ExclamationTriangleIcon />}
+        title="Could not retrieve operation overrides"
+        description={
+          data?.response?.details || error?.message || "Please try again"
+        }
+        actions={<Button onClick={() => refetch()}>Retry</Button>}
+      />
+    );
+
+  if (data.overrides.length === 0) {
+    return (
+      <EmptyState
+        icon={<InformationCircleIcon />}
+        title="No overrides found"
+        description="Overrides that you add from schema checks will appear here"
+      />
+    );
+  }
+
+  return (
+    <>
+      <TableWrapper>
+        <Table>
+          <TableCaption>
+            Found {data.overrides.length} operations with overrides
+          </TableCaption>
+          <TableHeader>
+            <TableRow>
+              <TableHead>Name</TableHead>
+              <TableHead>Hash</TableHead>
+              <TableHead>Change Overrides</TableHead>
+              <TableHead>Ignore Override</TableHead>
+              <TableHead>Last updated</TableHead>
+              <TableHead className="w-52"></TableHead>
+            </TableRow>
+          </TableHeader>
+          <TableBody>
```

Comment:
This works but it is the wrong approach to use reactTable. You use the hook `useReactTable` and render the table based on the table mode. Only in that way, reactable can leveraged to its full potential. See routers table.

Reply:
Please use the library as it is supposed to be. The next guy who needs to implement pagination, sorting, filtering thanks you.